### ISSUE TITLE
Uncaught TypeError: Cannot read property 'replace' of undefined

### ISSUE THREAD

**Issue by benogle** [jchouse/csscomb-atom#3]
_From @djdesign on December 11, 2014 5:52_

Uncaught TypeError: Cannot read property 'replace' of undefined

**Atom Version**: 0.156.0
**System**: Mac OS X 10.10.1
**Thrown From**: 'atom-css-comb' package, vundefined

### Steps To Reproduce

1. right clicks and selected Css Comb.

### Stack Trace

```
At /Users/DJ/.atom/packages/atom-css-comb/node_modules/csscomb/node_modules/csscomb-core/lib/core.js:319

TypeError: Cannot read property 'replace' of undefined
  at [object Object].processPath (/Users/DJ/.atom/packages/atom-css-comb/node_modules/csscomb/node_modules/csscomb-core/lib/core.js:319:20)
  at Object.module.exports.comb (/Users/DJ/.atom/packages/atom-css-comb/lib/css-comb.coffee:28:14)
  at atom-workspace.<anonymous> (/Users/DJ/.atom/packages/atom-css-comb/lib/css-comb.coffee:19:57)
  at atom-workspace.handler (/Applications/Atom.app/Contents/Resources/app/src/space-pen-extensions.js:112:34)
  at atom-workspace.jQuery.event.dispatch (/Applications/Atom.app/Contents/Resources/app/node_modules/space-pen/vendor/jquery.js:4681:9)
  at atom-workspace.elemData.handle (/Applications/Atom.app/Contents/Resources/app/node_modules/space-pen/vendor/jquery.js:4359:46)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app/src/command-registry.js:243:29)
  at CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app/src/command-registry.js:3:61)
  at CommandRegistry.module.exports.CommandRegistry.dispatch (/Applications/Atom.app/Contents/Resources/app/src/command-registry.js:156:19)
  at [object Object].jQuery.fn.trigger (/Applications/Atom.app/Contents/Resources/app/src/space-pen-extensions.js:64:23)
  at [object Object].$.fn.trigger (/Users/DJ/.atom/packages/command-logger/lib/main.coffee:35:15)
  at Ipc.<anonymous> (/Applications/Atom.app/Contents/Resources/app/src/window-event-handler.js:70:67)
  at Ipc.emit (events.js:110:17)
  at process.<anonymous> (/Applications/Atom.app/Contents/Resources/atom/renderer/api/lib/ipc.js:22:29)
  at process.emit (events.js:118:17)
  at Ipc.sendSync (/Applications/Atom.app/Contents/Resources/atom/renderer/api/lib/ipc.js:39:29)
  at Ipc.sendChannelSync (/Applications/Atom.app/Contents/Resources/atom/renderer/api/lib/ipc.js:47:28)
  at BrowserWindow.RemoteMemberFunction [as emit] (/Applications/Atom.app/Contents/Resources/atom/renderer/api/lib/remote.js:110:29)
  at ContextMenuManager.module.exports.ContextMenuManager.showForEvent (/Applications/Atom.app/Contents/Resources/app/src/context-menu-manager.js:169:33)
  at HTMLDocument.<anonymous> (/Applications/Atom.app/Contents/Resources/app/src/window-event-handler.js:138:33)
  at HTMLDocument.handler (/Applications/Atom.app/Contents/Resources/app/src/space-pen-extensions.js:112:34)
  at HTMLDocument.jQuery.event.dispatch (/Applications/Atom.app/Contents/Resources/app/node_modules/space-pen/vendor/jquery.js:4681:9)
  at HTMLDocument.elemData.handle (/Applications/Atom.app/Contents/Resources/app/node_modules/space-pen/vendor/jquery.js:4359:46)

```

_Copied from original issue: atom/atom#4533_

**Comment by Hurtak** [comment]
[Enter steps to reproduce below:]

1. I pressed right alt + c which prints '&' character on czech keyboard inside new empty untitled tab (i have 'ctrl-alt-c': 'unset!' in my keymap.cson).

**Atom Version**: 0.188.0
**System**: Unknown Windows Version
**Thrown From**: [atom-css-comb](https://github.com/jchouse/csscomb-atom) package, v0.6.4

### Stack Trace

Uncaught TypeError: Cannot read property 'replace' of undefined

```
At C:\Users\petr.hurtak\.atom\packages\atom-css-comb\node_modules\csscomb\node_modules\csscomb-core\lib\core.js:319

TypeError: Cannot read property 'replace' of undefined
  at [object Object].processPath (C:\Users\petr.hurtak\.atom\packages\atom-css-comb\node_modules\csscomb\node_modules\csscomb-core\lib\core.js:319:20)
  at Object.module.exports.comb (C:\Users\petr.hurtak\.atom\packages\atom-css-comb\lib\css-comb.coffee:28:14)
  at atom-workspace.atom.commands.add.css-comb:comb (C:\Users\petr.hurtak\.atom\packages\atom-css-comb\lib\css-comb.coffee:19:66)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (C:\Users\petr.hurtak\AppData\Local\atom\app-0.188.0\resources\app\src\command-registry.js:242:29)
  at CommandRegistry.handleCommandEvent (C:\Users\petr.hurtak\AppData\Local\atom\app-0.188.0\resources\app\src\command-registry.js:3:61)
  at CommandRegistry.module.exports.CommandRegistry.dispatch (C:\Users\petr.hurtak\AppData\Local\atom\app-0.188.0\resources\app\src\command-registry.js:157:19)
  at EventEmitter.<anonymous> (C:\Users\petr.hurtak\AppData\Local\atom\app-0.188.0\resources\app\src\window-event-handler.js:73:30)
  at EventEmitter.emit (events.js:116:17)

```

### Commands

```
     -1:12.7 editor:select-to-end-of-line (atom-text-editor.editor)
     -1:12.4 core:cut (atom-text-editor.editor)
  2x -1:12.2 core:move-up (atom-text-editor.editor)
  2x -1:11.9 block-travel:move-up (atom-text-editor.editor)
     -1:11.3 emmet:insert-formatted-line-break-only (atom-text-editor.editor)
     -1:11.3 docblockr:parse-enter (atom-text-editor.editor)
     -1:11.3 editor:newline (atom-text-editor.editor)
     -1:11.0 core:move-down (atom-text-editor.editor)
     -1:10.3 core:select-down (atom-text-editor.editor)
     -1:10.0 core:cut (atom-text-editor.editor)
     -1:01.3 editor:move-to-first-character-of-line (atom-text-editor.editor)
     -0:58.5 core:select-down (atom-text-editor.editor)
     -0:57.7 core:cut (atom-text-editor.editor)
     -0:12.8 editor:move-to-first-character-of-line (atom-text-editor.editor)
     -0:10.9 editor:move-to-end-of-screen-line (atom-text-editor.editor)
  2x -0:10.0 css-comb:comb (atom-text-editor.editor)
```

### Config

```json
{
  "core": {
    "themes": [
      "atom-dark-ui",
      "monokai"
    ],
    "disabledPackages": [
      "autocomplete-snippets"
    ],
    "followSymlinks": true,
    "destroyEmptyPanes": false
  }
}
```

### Installed Packages

```coffee
# User
atom-color-highlight, v3.0.8
atom-css-comb, v0.6.4
autocomplete-css, v0.6.0
autocomplete-html, v0.5.0
autocomplete-paths, v1.0.2
autocomplete-plus, v2.7.0
block-travel, v1.0.2
color-picker, v1.5.0
cursor-count, v0.2.0
docblockr, v0.6.3
editorconfig, v0.3.3
emmet, v2.3.4
highlight-selected, v0.9.1
last-cursor-position, v0.7.0
linter, v0.12.0
linter-csslint, v0.0.11
linter-htmlhint, v0.0.10
linter-jscs, v1.9.0
linter-jshint, v0.1.0
linter-jsonlint, v0.1.2
markdown-scroll-sync, v0.3.2
monokai, v0.13.0
pretty-json, v0.3.2
save-session, v0.12.4
sync-settings, v0.2.2
tabs-to-spaces, v0.9.1

# Dev
No dev packages
```

**Comment by ikeagold** [comment]
Easy to reproduce, create a new blank pane/tab and paste css code, select css syntax and try to run, you always got this error while you dont save this TEMPorary pane/tab to file (also i think its related to this case: csscomb must comb before we save file (edited state) too, because currently its working only on SAVED file, on edited - doing nothing).

**Comment by ikeagold** [comment]
@jchouse maybe it helps, but, beautifier can beautify before file saved.. maybe its belong to editor scope or filescope or how currently atom detect where he must apply toggled keybind (because as i know we must provide scope for it).

**Comment by jchouse** [comment]
Fixed in v3.0.0
